### PR TITLE
[15.0][IMP] mail_notification_custom_subject: add migration script for convert syntaxis of jinja2 to jinja3

### PR DESCRIPTION
**Pull request by ernesto-garcia-tecnativa**
@tecnativa TT42338

### CODE REVIEW

**Comment by ernesto-garcia-tecnativa** [comment]
@carlosdauden @pedrobaeza  please review

File: mail_notification_custom_subject/migrations/15.0.1.0.2/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# pylint: disable=C7902
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (pedrobaeza) [inline mail_notification_custom_subject/migrations/15.0.1.0.2/post-migration.py]: Don't mix licenses:

```suggestion
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

File: mail_notification_custom_subject/migrations/15.0.1.0.2/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if openupgrade.column_exists(
```
Comment (pedrobaeza) [inline mail_notification_custom_subject/migrations/15.0.1.0.2/post-migration.py]: This check is not adding value.

File: mail_notification_custom_subject/migrations/15.0.1.0.2/post-migration.py
Comment (pedrobaeza) [inline mail_notification_custom_subject/migrations/15.0.1.0.2/post-migration.py:1]: The folder should be `15.0.1.0.0`.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza please review

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — COMMENTED** [review]
:+1:

**Pull request merged**